packages/jest-console/src/getConsoleOutput.ts

```diff
@@ -11,18 +11,21 @@ import {
   StackTraceOptions,
   formatStackTrace,
 } from 'jest-message-util';
+import type {Config} from '@jest/types';
 import type {ConsoleBuffer} from './types';
 
 export default (
-  // TODO: remove in 26
+  // TODO: remove in 27
   root: string,
+  // TODO: this is covered by GlobalConfig, switch over in 27
   verbose: boolean,
   buffer: ConsoleBuffer,
-  // TODO: make mandatory and take Config.ProjectConfig in 26
+  // TODO: make mandatory and take Config.ProjectConfig in 27
   config: StackTraceConfig = {
     rootDir: root,
     testMatch: [],
   },
+  globalConfig: Config.GlobalConfig,
```

Comment:
Needs to be optional until next major ☹️ Should create an issue we can add to the Jest 27 milestone so we don't forget (again)

Reply:
I created [this one](https://github.com/facebook/jest/issues/10112) and can create a PR for it, please let me know if anything is missing.

Reply:
Perfect 👍